ktor-server/ktor-server-jetty/jvm/src/io/ktor/server/jetty/EngineMain.kt

```diff
@@ -10,6 +10,7 @@ import io.ktor.server.engine.*
 /**
  * Jetty engine
  */
+@Deprecated("This references an old version of Jetty. Use ktor-server-jetty-jakarta module instead")
```

Comment:
"Please consider changing dependency from 'ktor-server-jetty' to 'ktor-server-jetty-jakarta'"

Reply:
I suggest the following:
"The `ktor-server-jetty` module is deprecated and will be removed in the next major release as it references an outdated version of Jetty. Please use the `ktor-server-jetty-jakarta` module instead."

Reply:
We can replace next major release with `4.0.0` explicitly